lib/commonFunctions.js

```diff
@@ -137,6 +138,9 @@ exports.checkDeviceTime = function (currentDeviceTime, cfg, cb) {
 
       var FIFTEEN_MINUTES = 15 * 60 * 1000;
       if ( Math.abs(serverTime.valueOf()-deviceTime.valueOf()) > FIFTEEN_MINUTES ) {
+        if (rollbar) {
+          rollbar.info('Device time not set correctly or wrong timezone selected');
```

Comment:
I guess this is just the sort of thing that we'd generally have captured via a metric in the past. Is there a mechanism for determining where we log these things or are we moving toward Rollbar to capture this sort of stuff?

Reply:
I'm going to add it to Kissmetrics as I agree it seems like the right place to capture it. I think Brandon's argument is that since it's technically an error message, it should be in Rollbar too.

The time(s) are in the console log, which should be easy enough to find that I'm not sure how useful it would be to add it to the Rollbar object too.